### ISSUE TITLE
Searching and Sorting breaks if Google can't find the location

### ISSUE THREAD

**Issue by willans** [ethercreative/simplemap#48]
Searching and Sorting breaks when looping through entries if Google can't find the location.

So if I entered some gibberish (e.g. "dsfdsfdsfdsfds"), then it returns an error, rather than just no results.

It returns a `CDbException` error, complaining about `Unknown column 'distance' in 'order clause'.`

```
{% set locations = craft.entries.googleMap({
    location: 'dfdsfdsfdsdfsd',
    radius: 50,
    unit: 'mi'
}).order('distance') %}

{% for location in locations %}
<p>{{ location.title }} - {{ location.googleMap.distance }}</p>
{% endfor %}
```

**Comment by Tam** [comment]
Fixed in v1.6.2!

**Comment by willans** [comment]
Nice one, thanks 👍
